lxd/instances_get.go

```diff
@@ -292,7 +296,7 @@ func doInstancesGet(d *Daemon, r *http.Request) (interface{}, error) {
 			filteredProjects = []string{projectName}
 		}
 
-		result, err = tx.GetInstanceNamesByNodeAddress(filteredProjects, db.InstanceTypeFilter(instanceType))
+		result, err = tx.GetInstanceAndProjectNamesByNodeAddress(filteredProjects, db.InstanceTypeFilter(instanceType))
```

Comment:
I tend to call this `result` var something like `projectsInstances` to show what it contains.